python/kserve/kserve/logging.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import logging.config
+
+from .constants.constants import KSERVE_LOGLEVEL
+
+KSERVE_LOGGER_NAME = 'kserve'
+KSERVE_LOGGER_FORMAT = ('%(asctime)s.%(msecs)03d %(process)s %(name)s '
+                        '%(levelname)s [%(funcName)s():%(lineno)s] %(message)s')
+KSERVE_LOGGER_DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+logging.config.dictConfig({
+    'version': 1,
+    'disable_existing_loggers': False,
+    "formatters": {
+        "default": {
+            "()": "logging.Formatter",
+            "fmt": KSERVE_LOGGER_FORMAT,
+            "datefmt": KSERVE_LOGGER_DATE_FORMAT,
+        },
+    },
+    "handlers": {
+        "default": {
+            "formatter": "default",
+            "class": "logging.StreamHandler",
+            "stream": "ext://sys.stderr",
+        },
+    },
+    "loggers": {
+        KSERVE_LOGGER_NAME: {
+            "handlers": ["default"],
+            "level": KSERVE_LOGLEVEL,
```

Comment:
So here we can then use environment variable to set the log level right? Let’s add the documentation for this here https://kserve.github.io/website/master/modelserving/v1beta1/custom/custom_model/#environment-variables